.env.example

```diff
@@ -0,0 +1 @@
+PORT=8081
```

Comment:
Add an empty line at the end.

Refer this to let the VS code automatically does this job for you - https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
Done